app/client/components/dashboard/MasterWalletCard.jsx

```diff
@@ -31,7 +31,7 @@ class MasterWalletCard extends React.Component {
     const amountDisbursed = creditTransferStats.total_distributed / 100;
     const amountWithdrawn = creditTransferStats.total_withdrawn / 100;
     const amountReclaimed = creditTransferStats.total_reclaimed / 100;
-    const symbol = activeToken.symbol;
+    const symbol = activeToken && activeToken.symbol;
```

Comment:
We need to check `activeToken` exists across the app, as when you change org, it temporarily is undefined